index.js

```diff
@@ -174,20 +197,52 @@ async function smartContractInitialization() {
     const strDeployTx = await strDeploy.deploy(wallet.address, false, initialNodes, initialUrls, {gasLimit: 6000000} )
     const str = await strDeployTx.deployed()
     log(`NodeRegistry deployed at ${str.address}`)
-
-    //all TXs should now be confirmed:
-
     let nodes = await str.getNodes();
     log(`NodeRegistry nodes : ${JSON.stringify(nodes)}`)
 
-    log(`Added liquidity to uniswap exchange: ${formatEther(amt_token)} DATAcoin, ${formatEther(amt_token2)} OTHERcoin`)
     const ethwei = parseEther("1")
     let rate = await datatokenExchange.getTokenToEthInputPrice(ethwei)
     log(`1 DATAtoken buys ${formatEther(rate)} ETH`)
     rate = await othertokenExchange.getTokenToEthInputPrice(ethwei)
     log(`1 OTHERtoken buys ${formatEther(rate)} ETH`)
 
 
+/*
```

Comment:
Remove commented-out code?

Reply:
OK, just wanted a reference to the code from the ENS deployment script. I'll include a link to it